config/config-builder/src/key_manager_config.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use libra_config::config::{NodeConfig, SecureBackend, Token, VaultConfig};
+
+pub struct KeyManagerConfig {
+    json_rpc_endpoint: String,
+    rotation_period_secs: Option<u64>,
+    sleep_period_secs: Option<u64>,
+    template: NodeConfig,
+    txn_expiration_secs: Option<u64>,
+    vault_host: String,
+    vault_namespace: Option<String>,
+    vault_token: String,
+}
+
+impl Default for KeyManagerConfig {
+    fn default() -> Self {
+        let template = NodeConfig::default();
+        Self {
+            json_rpc_endpoint: template.secure.key_manager.json_rpc_endpoint.clone(),
+            rotation_period_secs: None,
+            sleep_period_secs: None,
+            template,
+            txn_expiration_secs: None,
+            vault_host: "".to_string(),
```

Comment:
@davidiw, the only thing I'm not certain about is defining defaults for vault_host and token? Sure, we could add garbage/test values for the host but what about the token? Is there really a default a token? (Similar in my mind to asking.. is there really a default private key? 😄)